.github/workflows/build_and_test.yml

```diff
@@ -39,3 +43,36 @@ jobs:
 
       - name: Build Electron app
         run: npm run do src/electron/build
+  build-and-test-windows:
+    name: Build and Test Web and Electron Apps on Windows
+    runs-on: ubuntu-20.04
+    steps:
+      - name : checkout
```

Comment:
Can these set up steps be put in a shared job that the two jobs depend on? I think you can use `needs`